Patches/RimEffect Asari/Asari_Race.xml

```diff
@@ -21,50 +21,51 @@
 					<li Class="PatchOperationAdd">
 					 <xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="Alien_Asari_Base"]/statBases</xpath>
 						  <value>
-						   <MeleeDodgeChance>0.20</MeleeDodgeChance>
-						   <MeleeCritChance>0.20</MeleeCritChance>
+							  <MeleeDodgeChance>1</MeleeDodgeChance>
+							  <MeleeCritChance>1</MeleeCritChance>
+							  <MeleeParryChance>1</MeleeParryChance>
 						  </value>
 					</li>
 					
 					<li Class="PatchOperationReplace">
 						<xpath>/Defs/AlienRace.ThingDef_AlienRace[defName="Alien_Asari_Base"]/tools</xpath> 
 						<value>
 							<tools>
-							  <li Class="CombatExtended.ToolCE">
+								<li Class="CombatExtended.ToolCE">
 									<label>left fist</label>
 									<capacities>
 										<li>Blunt</li>
 									</capacities>
-									<power>6.7</power>
-									<cooldownTime>1.65</cooldownTime>
+									<power>1</power>
+									<cooldownTime>1.4</cooldownTime>
 									<linkedBodyPartsGroup>LeftHand</linkedBodyPartsGroup>
-									<armorPenetration>0.09</armorPenetration>
-
-							  </li>
-							  <li Class="CombatExtended.ToolCE">
+									<armorPenetrationKPA>203</armorPenetrationKPA>

```

Comment:
armorPenetrationKPA has been renamed to armorPenetrationBlunt, and its value is now represented in MPa, so you should divide it by 1000.